tests/rptest/tests/cloud_storage_chunk_read_path_test.py

```diff
@@ -123,9 +124,13 @@ def _assert_files_in_cache(self, expr: str, must_be_absent=False):
             assert found_files, f'no files in cache dir matching expression {expr}'
 
     @cluster(num_nodes=4)
-    def test_read_chunks(self):
+    @parametrize(prefetch=0)
+    @parametrize(prefetch=3)
+    @parametrize(prefetch=10)
+    def test_read_chunks(self, prefetch):
         self._set_params_and_start_redpanda(
-            cloud_storage_cache_chunk_size=1048576 * 8)
+            cloud_storage_cache_chunk_size=1048576 * 8,
+            cloud_storage_chunk_prefetch=prefetch)
```

Comment:
Would be nice to check that prefetching is actually happening.

Reply:
added the test `tests.cloud_storage_chunk_read_path_test.CloudStorageChunkReadTest.test_prefetch_chunks` now, will require a few ci-repeats to see if it's parameters are correct.